<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Booking</class>
 <widget class="QDialog" name="Booking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bus Ticket Booking</string>
  </property>
  <widget class="QLabel" name="label_source">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Source:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sourceDropdown">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_destination">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="destinationDropdown">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Journey Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bookButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Ticket</string>
   </property>
  </widget>
  <widget class="QComboBox" name="seatNoDropdown">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_seat_number">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Seat Number:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
